app/src/main/java/org/coepi/android/domain/symptomflow/SymptomInputsManager.kt

```diff
@@ -45,6 +48,7 @@ class SymptomInputsManagerImpl :
                 COUGH -> acc.copy(cough = Cough())
                 BREATHLESSNESS -> acc.copy(breathlessness = Breathlessness())
                 FEVER -> acc.copy(fever = Fever())
+                EARLIESTSYMPTOM -> acc.copy(earliestSymptomDate = EarliestSymptom())
```

Comment:
Since it's not an id, not needed here. Besides, I think that `initInputs` is outdated and can be removed. Now the "top level" input classes are always set.